seisidd/devices/motors.py

```diff
@@ -9,66 +9,88 @@
-from ophyd   import EpicsMotor
-from ophyd   import EpicsSignal
-from ophyd   import EpicsSignalRO
+from ophyd   import    Device
+from ophyd   import    MotorBundle
+from ophyd   import    Component
+from ophyd   import    EpicsMotor
+from ophyd   import    EpicsSignal
+from ophyd   import    EpicsSignalRO
 
 
 class StageAero(MotorBundle):
     """
     Motor stacks used for HT-HEDM
 
-        _______________________________
-        | fine translation:  kx,ky,kz |
-        ===============================
-        | air-bearing rotation: rot   |
-        ===============================
-        | coarse translation: x, y, z |
-        -------------------------------
+        ___________________________________
+        |   fine translation:  kx,ky,kz   |
+        |   fine tilt: kx_tilt, kz_tilt   |
+        ===================================
+        |    air-bearing rotation: rot    |
+        ===================================
+        |  coarse translation below Aero: | 
+        |     x_base, y_base, z_base      |
+        -----------------------------------
 
-    TODO:
-        update with acutal PV
     """
-    kx_trans = Component(EpicsMotor, "$TRKX_PV", name='kx_trans')  # x motion with kozu stage
-    ky_trans = Component(EpicsMotor, "$TRKY_PV", name='ky_trans')  # y motion with kozu stage
-    kz_trans = Component(EpicsMotor, "$TRKZ_PV", name='kz_trans')  # z motion with kozu stage
 
-    rot_y    = Component(EpicsMotor, "$ROT_PV", name='rot_y'  )  # rotation with aero stage
-    x_trans  = Component(EpicsMotor, "$TRX_PV", name='x_trans')  # x motion with aero stage
-    y_trans  = Component(EpicsMotor, "$TRY_PV", name='y_trans')  # y motion with aero stage
-    z_trans  = Component(EpicsMotor, "$TRZ_PV", name='z_trans')  # z motion with aero stage
+    #   TODO:
+    #   update with acutal PV
+    kx      = Component(EpicsMotor, "$TRKX_PV", name='kx_trans')  # x motion with kohzu stage
+    ky      = Component(EpicsMotor, "$TRKY_PV", name='ky_trans')  # y motion with kohzu stage
+    kz      = Component(EpicsMotor, "$TRKZ_PV", name='kz_trans')  # z motion with kohzu stage
+    kx_tilt = Component(EpicsMotor, "$TTKX_PV", name='kx_tilt')   # kohzu tilt motion along x
+    kz_tilt = Component(EpicsMotor, "$TTKZ_PV", name='kz_tilt')   # kohzu tilt motion along z
+
+    rot     = Component(EpicsMotor, "$ROT_PV",  name='rot_y'  )    # rotation with aero stage
+
+    x_base  = Component(EpicsMotor, "$TRX_PV",  name='x_trans')    # x motion below aero stage
+    y_base  = Component(EpicsMotor, "$TRY_PV",  name='y_trans')    # y motion below aero stage
+    z_base  = Component(EpicsMotor, "$TRZ_PV",  name='z_trans')    # z motion below aero stage
 
     @property
     def status(self):
         """return full pv list and corresponding values"""
         # TODO:
-        #   once acutal PVs are known, the implementation should goes here
+        #   once acutal PVs are known, the implementation should go here
```

Comment:
use __repr__ instead of status()?  Or perhaps both?

Reply:
Let me investigate what this represents does...